### DISCUSSION TITLE
"Invalid username or password" when trying to load JVS Japanese "kan-bayashi/jvs_jvs010_vits_prosody" model

### DISCUSSION THREAD

**Discussion by RaduBolbo** [espnet/espnet#5461] [General]
I am using the script from the [guide](https://colab.research.google.com/github/espnet/notebook/blob/master/espnet2_tts_realtime_demo.ipynb#scrollTo=9e_i_gdgAFNJ) for TTS. When loading the "kan-bayashi/jvs_jvs010_vits_prosody" model, I get the following error: 

huggingface_hub.utils._errors.RepositoryNotFoundError: 401 Client Error. (Request ID: Root=1-651c1139-579fb23a19bc16cf7438a0b1;24ed3379-d186-44e8-9272-7fc847c0efba)

Repository Not Found for url: https://huggingface.co/api/models/jvs_jvs010_vits_prosody/revision/main.
Please make sure you specified the correct `repo_id` and `repo_type`.
If you are trying to access a private or gated repo, make sure you are authenticated.
Invalid username or password.

**Comment by Fhrozen** [comment]
Try with: https://huggingface.co/espnet/kan-bayashi_jvs_jvs010_vits_prosody or `espnet/kan-bayashi_jvs_jvs010_vits_prosody`
